packages/override-tools/src/Manifest.test.ts

```diff
@@ -0,0 +1,291 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import * as Manifest from './Manifest';
+
+import {
+  MockFile,
+  MockOverrideFileRepository,
+  MockReactFileRepository,
+} from './MockFileRepository';
+
+test('EmptyManifestInvalid', () => {
```

Comment:
>test [](start = 0, length = 4)

Is that common practice to place tests inside of src directory? Why not have a separate test directory?

Reply:
I've seen this both ways. Jest can also search for tests in a `__tests__` directory instead of resolving using `.test.ts`, but it will fail if you have any test helper code like mocks in there without real tests.

I think I can put things into a `tests` folder.